lib/server.c

```diff
@@ -90,7 +90,7 @@ static int _sasl_checkpass(sasl_conn_t *conn,
 
 static mech_list_t *mechlist = NULL; /* global var which holds the list */
 
-static sasl_global_callbacks_t global_callbacks;
+static sasl_global_callbacks_t server_global_callbacks;
```

Comment:
Client uses `global_callbacks_client`, should this follow the same schema?

Reply:
I'd be inclined to go the other way, since global_callbacks_server would make tab completion in gdb slower. As these are static variables, the only reason to rename them is to eliminate ambiguity in the debugger.

Reply:
I'll note that when this patch was written, this variable was a global and not static. So probably the patch was fixing an actual symbol collision before, and that's not an issue in the current code, so we could also just drop this commit.

Reply:
Dropped this commit